get-involved.php

```diff
@@ -195,10 +195,27 @@
         <div class="whole">
             <div class="actions">
                 <a class="button flat suggested-action" href="https://github.com/elementary" target="_blank" rel="noopener">Browse Our Code</a>
+            </div>
+        </div>
+        <div class="third">
+            <h3>Report issues</h3>
+            <p>Help out by tracking down issues and reporting those or by clarifying or cleaning up existing issues. <a class="read-more" href="/docs/code/reference#reporting-bugs" target="_blank" rel="noopener">Learn More</a></p>
```

Comment:
Suggest improved clarity/grammar:

`Help out by tracking down issues and reporting them, or by clarifying and cleaning up existing issues.`

Reply:
Thanks